packages/stream_chat_persistence/lib/src/db/shared/unsupported_db.dart

```diff
@@ -5,10 +5,11 @@ import 'package:stream_chat_persistence/stream_chat_persistence.dart';
 /// A Helper class to construct new instances of [DriftChatDatabase]
 class SharedDB {
   /// Returns a new instance of [DriftChatDatabase].
-  static DriftChatDatabase constructDatabase(
+  static Future<DriftChatDatabase> constructDatabase(
```

Comment:
hey @geweald sorry for the delay
If I'm not wrong this is breaking because this is now a future.

cc @xsahil03x

Reply:
Hey @imtoori well, this is only used in the `_defaultDatabaseProvider` (private) in the `StreamChatPersistenceClient` and `_defaultDatabaseProvider` is being called only in the `connect` which is a Future already, so this is not breaking anything

Reply:
yeah you're right, sorry about this
I'm talking about this with the team, I think we can merge it